### DISCUSSION TITLE
Install failed or worked partially, uninstall not working

### DISCUSSION THREAD

**Discussion by johannesgrundmann** [toltec-dev/toltec#755] [Support] [answered]
Hi guys,

I have a reMarkable 2 with v. 3.8.2.1965. Though I only afterwards learned that toltec eventually isn't compatible with my fw version, I tried to install toltec using the two lines from the documentation.

The first line "wget..." worked.

While pasting the second line "echo "04 ...", I accidentally added a c character, so the line was "echo "04 ... bash bootstrap**c**".  It then stopped with the message: "bootstrap: OK 
bash: bootstrapc: No such file or directory"

After that, neither the commands "opkg", nor "toltecctl" worked, so I can't uninstall using "toltecctl uninstall", because "toltecctl: command not found".

Also, trying to re-enter the install command, the "wget..." part seems still to work ("'bootstrap' saved" or "...file exists").

But the "echo..." part gives the following error: 
"bootstrap: OK
INFO:  Bootstrapping Opkg
bootstrap: line 119:   500 Segmentation fault      wget --no-verbose "$entware_remote/opkg" --output-document "$opkg_path"
ERROR: Could not fetch opkg, make sure you have a stable Wi-Fi connection"

Any tips on how to fully remove my failed toltec installation without bricking my device?

I am not that fit in linux and ssh :)

**Comment by johannesgrundmann** [comment]
@Eeems since you are the only one active here :-) or maybe I am overcomplicating and I should just remove opkg from .../local/bin and bootstrap from root?

**Comment by Eeems** [comment, marked as answer]
https://github.com/toltec-dev/toltec/issues/731

You haven't installed anything yet, so there is nothing to uninstall.

What tutorial were you following that didn't inform you to verify with the toltec site that your OS is supported?

**Comment by johannesgrundmann** [comment]
Thanks for answering :-)

Erm I actually followed the tutorial on https://toltec-dev.org/#install-toltec and just un-saw the prominent warning, so it's really my personal fault. 

However, can I just safely remove opkg and bootstrap files, while leaving the (original?) wget file?

**Reply by Eeems** [reply]
Yes you you can safely remove the bootstrap files, and if there is an opkg binary in /home/root/.local/bin, it is safe to remove as well.

**Reply by johannesgrundmann** [reply]
Ok thanks very much for your help, mate! Cheers from germany :)
